### PR TITLE
Add request-otr file to Local Data Files component

### PR DESCRIPTION
**Pull request by pilgrim-brave**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
...
+    "data-files-local-data-files": "npm install --prefix ./node_modules/brave-site-specific-scripts && npm run --prefix ./node_modules/brave-site-specific-scripts build && mkdir -p brave-lists && wget https://raw.githubusercontent.com/brave/adblock-lists/master/brave-lists/clean-urls.json -O brave-lists/clean-urls.json && wget https://raw.githubusercontent.com/brave/adblock-lists/master/brave-lists/debounce.json -O brave-lists/debounce.json && wget https://raw.githubusercontent.com/brave/adblock-lists/master/brave-lists/request-otr.json -O brave-lists/request-otr.json && wget https://raw.githubusercontent.com/brave/adblock-lists/master/brave-lists/https-upgrade-exceptions-list.txt -O brave-lists/https-upgrade-exceptions-list.txt && wget https://raw.githubusercontent.com/brave/adblock-lists/master/brave-lists/localhost-permission-allow-list.txt -O brave-lists/localhost-permission-allow-list.txt",
```
Comment (ShivanKaul) [inline package.json:33]: Doesn't really matter but is there a reason you didn't want to add this at the end?

**Comment by ShivanKaul** [comment]
@pilgrim-brave can you confirm that you get the expected list in `build/` when you run `npm run package-local-data-files -- --local-run`?

**Review by ShivanKaul — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (pilgrim-brave) [reply package.json:33]: Not really. I copy/pasted the debounce.json command because I know the OTR files are in the same directory.

**Comment by pilgrim-brave** [comment]
> @pilgrim-brave can you confirm that you get the expected list in `build/` when you run `npm run package-local-data-files -- --local-run`?

I do, yes.
Screenshot 2023-05-09 at 5 45 07 PM

**Pull request merged**